src/style.css

```diff
@@ -1 +1,58 @@
-/* Styles go here */
+.display {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  justify-content: end;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  gap: 10px;
```

Comment:
don't use gap, also the distance between menu items should be 20px, not 10px
add margin-right to nav__item, but remove it form the last element in the list